google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -1859,79 +1728,9 @@ public ApiFuture<QuerySnapshot> get() {
    * @return An ApiFuture that will be resolved with the planner information, statistics from the
    *     query execution (if any), and the query results (if any).
    */
-  @Nonnull
```

Comment:
Same here.

I think we still want this to be `@Nonnull` ?

Reply:
Same. Thanks.